src/extension.ts

```diff
@@ -86,57 +125,77 @@ export function activate(context: vscode.ExtensionContext) {
 				{
 					placeHolder: modeConfiguration.description,
 				},
-			).then(selectedMode => vscode.workspace.getConfiguration().update(modeConfigurationKey, selectedMode?.label))
+			).then(selectedMode => vscode.workspace.getConfiguration().update(ConfigurationKey.mode, selectedMode?.label));
 		}),
 		vscode.window.onDidChangeVisibleTextEditors(textEditors => {
-			hasSplitPanels = checkSplitPanels(textEditors)
-			showOrHideStatusBarItems()
+			hasSplitPanels = checkSplitPanels(textEditors);
+			showOrHideStatusBarItems();
 		}),
 		vscode.window.onDidChangeTextEditorVisibleRanges(({ textEditor, visibleRanges }) => {
 			if (!hasSplitPanels || !isOn) {
-				return
+				return;
 			}
 			if (scrollingEditor !== textEditor) {
 				if (scrolledEditorsQueue.has(textEditor)) {
-					scrolledEditorsQueue.delete(textEditor)
-					return
+					scrolledEditorsQueue.delete(textEditor);
+					return;
 				}
-				scrollingEditor = textEditor
-				if (vscode.workspace.getConfiguration().get(modeConfigurationKey) === 'OFFSET') {
+				scrollingEditor = textEditor;
+				if (vscode.workspace.getConfiguration().get(ConfigurationKey.mode) === 'OFFSET') {
 					vscode.window.visibleTextEditors
 						.filter(editor => editor !== textEditor)
 						.forEach(scrolledEditor => {
-							offsetByEditors.set(scrolledEditor, scrolledEditor.visibleRanges[0].start.line - textEditor.visibleRanges[0].start.line)
-						})
+							offsetByEditors.set(scrolledEditor, scrolledEditor.visibleRanges[0].start.line - textEditor.visibleRanges[0].start.line);
+						});
 				}
 			}
 			if (scrollingTask) {
-				clearTimeout(scrollingTask)
+				clearTimeout(scrollingTask);
 			}
 			scrollingTask = setTimeout(() => {
 				vscode.window.visibleTextEditors
 					.filter(editor => editor !== textEditor)
 					.forEach(scrolledEditor => {
-						scrolledEditorsQueue.add(scrolledEditor)
+						scrolledEditorsQueue.add(scrolledEditor);
 						scrolledEditor.revealRange(
 							calculateRange(visibleRanges[0], offsetByEditors.get(scrolledEditor) ?? 0, textEditor, scrolledEditor),
 							vscode.TextEditorRevealType.AtTop,
-						)
-					})
-			}, 0)
+						);
+					});
+			}, 0);
 		}),
 		vscode.workspace.onDidChangeConfiguration(({ affectsConfiguration }) => {
-			if (affectsConfiguration(modeConfigurationKey)) {
-				reset()
-				updateStatusBarMode()
+			if (affectsConfiguration(ConfigurationKey.mode)) {
+				reset();
+				updateStatusBarMode();
+			}
+			if (affectsConfiguration(ConfigurationKey.saveRecentToggleState)) {
+				saveRecentToggleState(context, isOn);
 			}
 		}),
-	)
+	);
 
 	// Init
-	toggleOn()
-	showOrHideStatusBarItems()
-	updateStatusBarMode()
+	// TODO: forDebug: Delete later
```

Comment:
You should remove these comments before merging so that it clean @k-ta-yamada

Reply:
👍